libs/blocks/global-navigation/blocks/profile/milo-wrapper.js

```diff
@@ -0,0 +1,39 @@
+import { toFragment, getPlaceholders } from '../../utilities.js';
+
+const initProfileButton = async ({
+  blockEl,
+  decoratedEl,
+  avatar,
+}) => {
+  const placeholders = await getPlaceholders(['profile-button']);
```

Comment:
Why `getPlaceholders` if we just need one?

Reply:
Frankly, didn't want to import it if we already have `getPlaceholders` - but there's no downside so I'll switch it up 😁

Reply:
Done!